pyshtools/shclasses/shcoeffs.py

```diff
@@ -4215,10 +4234,10 @@ def _rotate(self, angles, dj_matrix):
                 "'unnorm'. Input value is {:s}."
                 .format(repr(self.normalization)))
 
-        coeffs_rot = _shtools.SHExpandDHC(grid_rot, norm=norm,
+        coeffs_rot = backend().SHExpandDHC(grid_rot, norm=norm,
```

Comment:
See comment above. I think that we should probably keep `_shtools` for consistency and since we are forcing to always use shtools in this code block.